MODULE.bazel

```diff
@@ -393,3 +393,12 @@ git_override(
     commit = "db677011c7363509a288a9fb3bf0a50830bbf791",
     remote = "https://github.com/erenon/bazel_clang_tidy.git",
 )
+
+http_file = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+http_file(
+    name = "minio",
+    executable = True,
+    integrity = "sha256-jsQvcPXbe/5YR5CwavP34rFvSEiOpMluDr2M6kCZDfo=",
+    urls = ["https://dl.min.io/server/minio/release/linux-amd64/minio"],
```

Comment:
I guess no one does dev on arm so amd only is fine

Reply:
yeh, good point. mmm, yeh id happily change this if we have an arm developer